app/serializers/user_serializer.rb

```diff
@@ -282,7 +282,7 @@ def ignored
   end
 
   def can_ignore_user
-    SiteSetting.ignore_user_enabled? && !object.staff? && scope.current_user != object
+    SiteSetting.ignore_user_enabled? && !object.staff? && scope.current_user != object && (scope.current_user.staff? || scope.current_user.trust_level >= 2)
```

Comment:
That line is getting too long. Might want to put each conditions on its own line 😉 

Also, can you use `TrustLevel.levels[:member]` instead of the magic number `2`?

Reply:
Makes sense 👍